<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExample">
    <property name="title">
     <string>Example</string>
    </property>
    <addaction name="actionDefault"/>
    <addaction name="actionSimple_Object"/>
    <addaction name="actionColored_quad"/>
    <addaction name="actionOctahedron"/>
    <addaction name="actionTessellator"/>
    <addaction name="actionTriangle_Faces"/>
   </widget>
   <addaction name="menuExample"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionSimple_Object">
   <property name="text">
    <string>Simple Object</string>
   </property>
  </action>
  <action name="actionColored_quad">
   <property name="text">
    <string>Colored quad</string>
   </property>
  </action>
  <action name="actionOctahedron">
   <property name="text">
    <string>Octahedron</string>
   </property>
  </action>
  <action name="actionTessellator">
   <property name="text">
    <string>Tessellator</string>
   </property>
  </action>
  <action name="actionTriangle_Faces">
   <property name="text">
    <string>Triangle Faces</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
